package.json

```diff
@@ -44,10 +44,13 @@
     "cheerio": "0.22.0",
     "colors": "1.1.2",
     "commander": "2.9.0",
+    "emoji-name-map": "^1.2.2",
```

Comment:
This is pretty much a wrapper for `emojilib` package. Said package also includes a `simplemap.json` file, which could be a better choice to use, as it doesn't use the full emojis.json.